Source/Core/DolphinQt/DolphinQt.vcxproj

```diff
@@ -473,10 +482,10 @@
     <ProjectReference Include="$(CoreDir)VideoBackends\D3D\D3D.vcxproj">
       <Project>{96020103-4ba5-4fd2-b4aa-5b6d24492d4e}</Project>
     </ProjectReference>
-    <ProjectReference Include="$(CoreDir)VideoBackends\OGL\OGL.vcxproj">
+    <ProjectReference Include="$(CoreDir)VideoBackends\OGL\OGL.vcxproj" Condition="'$(Platform)'!='ARM64'">
```

Comment:
I get the feeling you wrote this one by hand, while all the others are VS generated. All the others could be more compact if they just did a Platform check instead of a combined Platform/Configuration one; but VS won't do that using the UI.

Reply:
Done.